src/app/website/course-home-page/why-learn/why-learn.component.ts

```diff
@@ -0,0 +1,67 @@
+import { Component, OnInit } from '@angular/core';
+import { Http, Response } from '@angular/http';
+import 'rxjs/Rx';
+import { ContentfulService } from '../../shared/contentful/contentful.service';
+
+
+@Component({
+  selector: 'tf-why-learn',
+  templateUrl: './why-learn.component.html',
+  styleUrls: ['./why-learn.component.scss']
+})
+export class WhyLearnComponent implements OnInit {
+  contentData = [];
+
+
+  constructor(private http: Http,
+    private contnetfulService: ContentfulService) { }
+
+  ngOnInit() {
+    this.getContents();
+
+  }
+
+
+  getContents() {
+    this.contnetfulService.getWhyLearnWithUsData()
+      .subscribe(
+      (data: any) => {
+
+
+        for (let item of data.items) {
```

Comment:
use functional programming and not for loop. Array has a function forEach. You should se that.